pkg/imgpkg/cmd/pull.go

```diff
@@ -130,5 +130,9 @@ func (po *PullOptions) validate() error {
 	if po.BundleRecursiveFlags.Recursive && len(po.ImageFlags.Image) > 0 {
 		return fmt.Errorf("Cannot use --recursive (-r) flag when pulling a bundle")
 	}
+
+	if !po.ImageIsBundleCheck && len(po.BundleFlags.Bundle) != 0 {
+		return fmt.Errorf("Setting --image-is-bundle-check flag to false is not allowed when using -b flag.")
```

Comment:
Should we just say that the flag cannot be used while using `-b`? The default value of the flag is true, so technically that will always be set, but let's not confuse users by saying that they can set it to true but cannot set it to false.

Reply:
Updated